# Rover Speed control for Teleoperation
Header header
float64 xSpeed
float64 ySpeed
float64 rotationAngle